<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBConfirmationDialog</class>
 <widget class="BBConfirmationDialog" name="ConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: rgb(14, 19, 25);
}

#labelTitle {
    color: #d6dfeb;
    font: 10pt &quot;Arial&quot;;
}

QPushButton {
    border: none;
    border-radius: 6px;
    color: #d6dfeb;
    font: 9pt &quot;Arial&quot;;
}

#buttonClose {
    background-color: rgb(252, 98, 93);
}

#buttonClose:pressed {
    background-color: rgb(239, 78, 73);
}

#bodyWidget {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #191f28, stop: 1 rgb(36, 43, 58));
}

#labelIcon {
    image: url(://resources/icons/notice.png);
}

#scrollAreaWidgetContents { /*The background color of the scroll area*/
    background: transparent;
}

QScrollArea { /*When both the horizontal and vertical scroll bars appear, the background color of the small square in the lower right corner*/
    background: transparent;
    border: none;
}

#labelMessage {
    color: #d6dfeb;
    font: 9pt &quot;Arial&quot;;
}

#buttonNo {
    border-radius: 2px;
    padding: 2px 12px;
    background: #8193bc;
}

#buttonNo:hover {
    background: #8c8193bc;
}

#buttonYes {
    border-radius: 2px;
    padding: 2px 12px;
    background: #0ebf9c;
}

#buttonYes:hover {
    background: #8c0ebf9c;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="topWidget">
     <layout class="QHBoxLayout" name="topWidgetLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="buttonClose">
        <property name="minimumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14</width>
          <height>14</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Notice</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bodyWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="bodyWidgetLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="text">
         <string/>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
          <horstretch>3</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <layout class="QVBoxLayout" name="scrollAreaLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelMessage">
            <property name="text">
             <string>This is Message.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignBottom | Qt::AlignRight">
    <widget class="QWidget" name="bottomWidget">
     <layout class="QHBoxLayout" name="bottomWidgetLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonNo">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonYes">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>BBConfirmationDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>buttonNo</sender>
   <signal>clicked()</signal>
   <receiver>BBConfirmationDialog</receiver>
   <slot>reject()</slot>
  </connection>
  <connection>
   <sender>buttonYes</sender>
   <signal>clicked()</signal>
   <receiver>BBConfirmationDialog</receiver>
   <slot>accept()</slot>
  </connection>
 </connections>
</ui>

